src/crimson/osd/osdmap_gate.h

```diff
@@ -46,8 +46,7 @@ class OSDMapGate {
   // so we can access all the waiters expecting a map whose epoch is less
   // than a given epoch
   using waiting_peering_t = std::map<epoch_t,
-				     OSDMapBlocker,
-				     std::greater<epoch_t>>;
+				     OSDMapBlocker>;
```

Comment:
could you note down the rationales of this change in the commit message?

Reply:
This is right, I have no idea why we set it to be in descending order.  got_map releases the blocked ops on [begin(), upper_bound(epoch)).  Fix the comment as well.

Reply:
my bad.. i thought the other way around in 3076020a82cc6ea441998cfa145968659baf7090.